internify/src/components/organisms/styles/AuthNavbar.css

```diff
@@ -0,0 +1,65 @@
+.auth_navbar_container {
+
```

Comment:
install the `Prettier` extension on VSCode and run it before you commit the next time so that it formats the code nicely

Reply:
thanks for the heads up Tommy - will do